### ISSUE TITLE
Only 1 user can connect to my MTProxy! Please help

### ISSUE THREAD

**Issue by Millidan** [TelegramMessenger/MTProxy#129]
Hello, I have my MTProxy running with 50 workers but I can only connect with one device, when that device is connected the Proxy shows as "Connecting" on any other device trying to connect. What am I doing wrong? I've been trying to fix this for 2 days but I can't!

**Comment by Eftekhar** [comment]
Hello

Edit this line:
./mtproto-proxy -u nobody -p 8888 -H 443 -S <secret> --aes-pwd proxy-secret proxy-multi.conf -M **1**

change **1** in this command

**Comment by Millidan** [comment]
smh

**Comment by gggin** [comment]
A Question, Only 30 worker that I can run, so I can only support 30 People in the same time?

**Comment by gggin** [comment]
A Question, Only 30 worker that I can run, so I can only support 30 People in the same time?

**Comment by gggin** [comment]
A Question, Only 30 worker that I can run, so I can only support 30 People in the same time?

**Comment by HamedFayez** [comment]
@gggin no. you can support so many people, as long as your server can handle.

**Comment by gggin** [comment]
@HamedFayez  Thank you for solve my problem.

**Comment by HamedFayez** [comment]
every worker have one thread, so if you have multi-core cpu you can use multiple workers to use all system process performance.

**Comment by HamedFayez** [comment]
@kooran if you use greater number of workers(= Threads) than the ability of your CPU , it will reduce its performance. i suggest you use 16 Workers (if your cpu have 16 real threads).

**Comment by samanb1990** [comment]
hi @HamedFayez 
im having the same issue
i want to connect more clients to my MTProto and i want to share it
how can i increase the number of worker?

 https://github.com/alexbers/mtprotoproxy   ===>   in this repository it says "The proxy performance should be enough to comfortably serve about **4 000** simultaneous users on the VDS instance with **1 CPU core** and **1024MB RAM**."

so the question is how to do that?
